azext_iot/iothub/commands_state.py

```diff
@@ -0,0 +1,26 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+from azext_iot.iothub.providers.job import JobProvider
+from azext_iot.iothub.providers.state import StateProvider
+
+import time
+
+logger = get_logger(__name__)
+
+
+def state_export(cmd, hub, rg=None, filename='/project/files/deviceFile.json', auth_type_dataplane=None):
+    sp = StateProvider(cmd, hub, rg, auth_type_dataplane=None)
```

Comment:
Apart from auth_type_dataplane, should we also support connection strings for all the commands?